plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -955,8 +956,27 @@ private void finishOptimize(ConnectorSession session, IcebergTableExecuteHandle
                             .map(dataFile -> dataFile.path().toString())
                             .collect(toImmutableSet()));
         }
+        // The idea is that after optimize there should not be any delete files used anymore as
```

Comment:
Don't start comment with "The idea is that".
Be sure (or appear to be sure) that you know what the code is doing.

Reply:
Hmm by writing the idea is that, I am trying to explain the concept of the solution not prove why it is correct.
But anyway thanks I think I 've got it.